_posts/2021-04-25-ocp-java11.md

```diff
@@ -0,0 +1,225 @@
+who order profiles of consultants it is obvious from reading mine that I have a rock solid project experience and
+whether I have the certificate or not does not really matter. 
+
+So why should I study even basic things again?
+
+It began with an initiative from a colleague of mine at Karakun. He made us aware in our internal channels that Oracle had an offer 
+to do the OCP Java 11 Developer exam for just $25, when taking the exam before April 25th 2021. This special price offer was because 
+of the 25 year anniversary of Java. So he asked who would be interested to create a working group to do the exam. 
+He himself did already a Java certification before. As a Java centric consulting company, several colleagues have done a certification before. 
+However, no-one had the Java 11 certification so far, and Java did evolve quite much in the recent years. 
+For him, as for me, one main motivation was to dig deeper in the more recent features, which are not all in use in our daily work. 
+Some of our newer colleagues joined the working group.
+
+The goal was not just to pass the exam, but really took the chance to study, and push us together to achieve the goal.
+
+## The Study Phase
+
+We started end of February and decided to learn either using the [OCP Oracle Certified Professional Java SE 11 Developer Complete Study Guide 
+by Jeanne Boyarsky and Scott Selikoff](https://www.selikoff.net/ocp11-complete/), or using the Oracle video tutorials. The book is 1300 pages, the videos 33 hours. 
+Quite a bunch! For the book we've set up a plan until when we would try to work through the chapters. The schedule we set up was:
+
+<figure class="wp-block-table"><table><thead><tr><th scope="col">Week</th><th scope="col">Dates</th><th scope="col">Chapters</th><th scope="col">Num pages</th></tr></thead><tbody><tr><td>11</td><td>15-21 March</td><td>1 Welcome to Java<br>2 Java Building Blocks<br>3 Operators<br>4 Making Decisions<br>5 Core Java APIs</td><td>~220</td></tr><tr><td>12</td><td>22-28 March</td><td>6 Lambdas and Functional Interfaces<br>7 Methods and Encapsulation<br>8 Class Design<br>9 Advanced Class Design</td><td>~180</td></tr><tr><td>13</td><td>29-4 April</td><td>10 Exceptions<br>11 Modules<br>12 Java Fundamentals<br>13 Annotations</td><td>~200</td></tr><tr><td>14</td><td>5-11 April</td><td>14 Generics and Collections<br>15 Functional Programming<br>16 Exceptions, Assertions, and Localization</td><td>~200</td></tr><tr><td>15</td><td>12-18 April</td><td>17 Modular Applications<br>18 Concurrency<br>19 I/O<br>20 NIO.2</td><td>~220</td></tr><tr><td>16</td><td>19-25 April</td><td>21 JDBC<br>22 Security</td><td>~100</td></tr></tbody></table></figure>
+
+At Karakun we have an education budget that we can use on our behalf, but to be honest,
+preparing such an exam also involves a high degree of spare time. We are also quite busy with our daily projects, and to take time it is not 
+always the right time in a project.
```

Comment:
> to take time it is not always the right time in a project
This is not very understandable. Did you mean "taking time is not always possible" ?

Reply:
yes, meant that. rephrased the paragraph.